rpc/http.go

```diff
@@ -212,17 +206,17 @@ func httpTimestamp(t time.Time) []byte {
 	t = t.UTC()
 	yy, mm, dd := t.Date()
 	hh, mn, ss := t.Clock()
-	day := days[3 * t.Weekday():]
```

Comment:
Not directly related to this PR, but why does this function even exist?
The only use of this function can basically be replaced by `time.Now().Format(http.TimeFormat)`.